### ISSUE TITLE
Need a docker compose example

### ISSUE THREAD

**Issue by kevintanhongann** [tensorchord/pgvecto.rs#438]
Need a docker compose example for pgvector.rs

**Comment by gaocegege** [comment]
Hi, here is an example https://github.com/immich-app/immich/blob/main/docker/docker-compose.yml#L66:#L75

Does it work for you?

**Comment by kevintanhongann** [comment]
@gaocegege I'm going to find out because I'm in the middle of experimenting Spring AI and pgvector, and because pgvector has a dimension size limitation

**Comment by gaocegege** [comment]
Yeah, you could check out our documentation for OpenAI 3072 dimension embedding model if you are interested.

https://docs.pgvecto.rs/use-case/adaptive-retrieval.html

**Comment by gaocegege** [comment]
@kevintanhongann, have you encountered any difficulties in your investigation of Spring AI and pgvector/pgvecto.rs? We are more than willing to assist you.
